### PR TITLE
Update New-CsCallQueue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updating for HideAuthorizedUsers parameter

### CODE REVIEW

**Comment by CLYVR** [comment]
@get-itips @dariomws @tseward @tiburd Please and thanks!

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 1]
```diff
@@ -1183,6 +1183,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -HideAuthorizedUsers
+This is a list of GUIDs of AuthorizedUsers who should not appear on the list of supervisors for the agents who are members of this queue.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallQueue.md:1187]:
```suggestion
This is a list of GUIDs of authorized users who should not appear on the list of supervisors for the agents who are members of this queue.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 2]
```diff
@@ -1200,6 +1200,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -HideAuthorizedUsers
+This is a list of GUIDs of AuthorizedUsers who should not appear on the list of supervisors for the agents who are members of this queue.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsCallQueue.md:1204]:
```suggestion
This is a list of GUIDs of authorized users who should not appear on the list of supervisors for the agents who are members of this queue.  The GUID should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CLYVR, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by tiburd** [comment]
@CLYVR 
Hi Colin,
There's been a change in who reviews PowersShell PRs--for future reference, the reviewers are get-itips and msbemba.
Thanks!

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**